packages/helm/symphony/files/oss/delete-objects.sh

```diff
@@ -1,29 +1,117 @@
 #!/bin/env bash
 set -e
-TIMEOUT="30m"
```

Comment:
We also have delete-objects.sh for files/azure, do we still use that? If we are not sure, please write changes on two version at first.